src/store/ui/actions.ts

```diff
@@ -194,7 +205,7 @@ export const startBuySellMarketSteps: ThunkCreator = (amount: BigNumber, side: O
         const baseTokenBalance = selectors.getBaseTokenBalance(state);
 
         const orders = side === OrderSide.Buy ? selectors.getOpenSellOrders(state) : selectors.getOpenBuyOrders(state);
-        const { amounts, canBeFilled } = buildMarketOrders(
+        const [, filledAmounts, canBeFilled] = buildMarketOrders(
```

Comment:
instead of leading with the comma, can we just assign the first return value to `_ordersToFill`?